rct229/rulesets/ashrae9012019/section1/section1rule3.py

```diff
@@ -0,0 +1,154 @@
+from pydash import filter_
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.ruleset_model_factory import produce_ruleset_model_instance
+from rct229.rulesets.ashrae9012019 import (
+    BASELINE_0,
+    BASELINE_90,
+    BASELINE_180,
+    BASELINE_270,
+    PROPOSED,
+    USER,
+)
+from rct229.utils.assertions import assert_
+from rct229.utils.jsonpath_utils import find_one
+from rct229.utils.std_comparisons import std_equal
+
+
+class Section1Rule3(RuleDefinitionBase):
+    """Rule 3 of ASHRAE 90.1-2019 Appendix G Section 1 (Performance Calculations)"""
+
+    def __init__(self):
+        super(Section1Rule3, self).__init__(
+            rmrs_used=produce_ruleset_model_instance(
+                USER=True,
+                BASELINE_0=True,
+                BASELINE_90=True,
+                BASELINE_180=True,
+                BASELINE_270=True,
+                PROPOSED=True,
+            ),
+            rmrs_used_optional=produce_ruleset_model_instance(
+                USER=True,
+                BASELINE_0=True,
+                BASELINE_90=True,
+                BASELINE_180=True,
+                BASELINE_270=True,
+                PROPOSED=True,
```

Comment:
Remove user, baseline_0 and proposed or set them to False.

Reply:
I tested this out and decided to leave this as is b/c ```USER```, ```BASELINE_0```, and ```PROPOSED``` are also optional for this ruleset.